package.json

```diff
@@ -25,7 +25,7 @@
     "fix:stylelint": "stylelint '**/*.css' --fix",
```

Comment:
I see we are missing the last changes from dashboard  https://github.com/awslabs/iot-app-kit/commit/f6a25afce0ebf0b87c0dd9d40c82a8326301130e

Without this we are forced to create a declaration file with the dashboard module in a consuming package.